browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -137,6 +138,12 @@ void BraveToolbarView::Init() {
       kShowBookmarksButton, browser_->profile()->GetPrefs(),
       base::BindRepeating(&BraveToolbarView::OnShowBookmarksButtonChanged,
                           base::Unretained(this)));
+
+  show_side_panel_button_.Init(
+      kShowSidePanelButton, browser_->profile()->GetPrefs(),
+      base::BindRepeating(&BraveToolbarView::OnShowSidePanelButtonChanged,
+                          base::Unretained(this)));
```

Comment:
How about calling UpdateSidePanelButtonVisibility() here so that we can initialize its visibility?

Reply:
:+1: good call. I think this means I can get rid of the call in `Update` too, which makes the code a bit simpler.

Reply:
@fallaciousreasoning are these calls suppose to differ slightly? how about refactoring it a little and make both `Update*Visibility` calls work similarly?

Reply:
Sorry @goodov, I'm not entirely sure what you mean here. `UpdateSidePanelButtonVisibility` is intended to update just the side panel button, in response to the pref change, and initialization. It works almost identically to similar methods, like `UpdateBookmarkVisibility`